.werft/jobs/build/job-config.ts

```diff
@@ -113,6 +114,7 @@ export function jobConfig(werft: Werft, context: any): JobConfig {
 
     const analytics = parseAnalytics(werft, sliceId, buildConfig["analytics"])
     const withIntegrationTests = parseWithIntegrationTests(werft, sliceId, buildConfig["with-integration-tests"]);
+    const withLatestIDEVersion = "with-latest-ide-version" in buildConfig;
```

Comment:
@mads-hartmann @iQQBot I would like to run to verify
```
/werft run with-integration-tests=jetbrains with-large-vm with-preview with-latest-ide-version
```

but my understanding that first it should be on main, otherwise it won't pick up `with-latest-ide-version` ?

Reply:
I can run it from workspace with werft.